js/src/options.ts

```diff
@@ -28,4 +29,13 @@ export type TranslationOptions = {
     language: string;                         // Set the translation language, e.g. fr-FR, es-MX, zh-Hans-CN. Required to automatically enable word or document translation.
     autoEnableDocumentTranslation?: boolean;  // Automatically translate the entire document
     autoEnableWordTranslation?: boolean;      // Automatically enable word translation
-};
\ No newline at end of file
+};
+
+export type InternalOptions = {
+    messageOptions?: InternalOptionDictionary   // Reserved for internal use. Additional options to be sent through Content message.
```

Comment:
Since you don't actually use the `InternalOptionDictionary` type anywhere beyond these two lines, you can just simplify these lines to: `messageOptions?: [option: string]: boolean | string`